### DISCUSSION TITLE
Sandıkta sıkıntı var galiba

### DISCUSSION THREAD

**Discussion by cihanaslan0** [IBAX-io/go-ibax#136] [Bug Submission]
Sandık buga girmiş galiba insanlar binlerce puan kasmış

**Comment by cartvecurt** [comment]
Olsa bile çözümü kolay, sandıktan önceki haline getirirler puanları, sandık hakkı verirler tekrar

**Comment by hemresenol** [comment]
Sistem böyle şeyleri kayıt altınta tutar puanları istediği kadar kassınlar bug gidince aldıkları puan gözükür
